app/routes/SafesList/components/Layout.js

```diff
@@ -9,18 +9,26 @@ class Layout extends Component {
     const {
       safes,
       selectSafe,
+      lockedState,
     } = this.props
 
     return (
-      <Page account logOut>
-        <Link to={{
-          pathname: '/password',
-          state: {
-            dest: '/pairing'
-          }
-        }}>
-          <button>Connect to safe</button>
-        </Link>
+      <Page account logOut padding='noPadding'>
+        {lockedState
+          ?
+          <Link to={{
```

Comment:
If the only difference is the ```to``` object I suggest to only evaluate it separately. Something like:
```
<Page>
    <Link to={ lockedState ? passwordRoute : pairingRoute }>
        <button> .... </button>
    </Link>
</Page>
```